central/sensor/service/pipeline/imageintegrations/pipeline.go

```diff
@@ -139,11 +170,11 @@ func setUpIntegrationParams(ctx context.Context, imageIntegration *storage.Image
 	switch config := imageIntegration.GetIntegrationConfig().(type) {
 	case *storage.ImageIntegration_Docker:
 		dockerCfg := config.Docker
-		var validTLS bool
-		validTLS, err = tlscheck.CheckTLS(ctx, dockerCfg.GetEndpoint())
-		if err != nil {
-			err = errors.Wrapf(err, "reaching out for TLS check to %s", dockerCfg.GetEndpoint())
-			return
+		validTLS, tlsErr := tlscheck.CheckTLS(ctx, dockerCfg.GetEndpoint())
+		if tlsErr != nil {
+			log.Debugf("reaching out for TLS check to %s: %v", dockerCfg.GetEndpoint(), err)
+			// Assume TLS is true
+			validTLS = true
 		}
 		dockerCfg.Insecure = !validTLS
```

Comment:
```suggestion
		if tlsErr != nil {
			log.Debugf("reaching out for TLS check to %s: %v", dockerCfg.GetEndpoint(), err)
			// Not enough evidence that we can skip TLS, so conservatively require it
			dockerCfg.Insecure = false
		} else {
			dockerCfg.Insecure = !validTLS
		}
```
Don't overload variables if the name doesn't fit the usage any more. `validTLS` is a misnomer in l.177.